recipes/utm/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "utm" %}
+{% set version = "0.4.2" %}
+{% set sha256 = "a6608a67df84418fd959a79b228b90ab55b2ae877827f9c210947104c5a75d0e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  run:
+    - python
+
+test:
+  imports:
+    - utm
+
+about:
+  home: https://github.com/Turbo87/utm
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Bidirectional UTM-WGS84 converter for python.'
+  description: 'Bidirectional UTM-WGS84 converter for python.'
+  doc_url: https://github.com/Turbo87/utm
+  dev_url: https://github.com/Turbo87/utm
+
+extra:
+  recipe-maintainers:
+    - snowman2
+    - Turbo87
```

Comment:
@Turbo87 -- are you ok with being added as a co-maintainer of the utm conda package on conda-forge?